Doc/c-api/long.rst

```diff
@@ -85,13 +85,12 @@ All integers are implemented as "long" integer objects of arbitrary size.
    Return a new :c:type:`PyLongObject` based on the string value in *str*, which
    is interpreted according to the radix in *base*.  If *pend* is non-*NULL*,
    *\*pend* will point to the first character in *str* which follows the
-   representation of the number.  If *base* is ``0``, the radix will be
-   determined based on the leading characters of *str*: if *str* starts with
-   ``'0x'`` or ``'0X'``, radix 16 will be used; if *str* starts with ``'0o'`` or
-   ``'0O'``, radix 8 will be used; if *str* starts with ``'0b'`` or ``'0B'``,
-   radix 2 will be used; otherwise radix 10 will be used.  If *base* is not
-   ``0``, it must be between ``2`` and ``36``, inclusive.  Leading spaces are
-   ignored.  If there are no digits, :exc:`ValueError` will be raised.
+   representation of the number.  If *base* is ``0``, the *str* is interpreted using
+   the :ref:`integers` defintion; additionally, a literal 0 evaluates as *base* 0 and
+   a :attr:`decinteger` beginning with 0 raises a :exc:`ValueError`.  If *base* is not
```

Comment:
Thank you!

The 'literal 0 evaluates as *base* 0 was for the case mentioned in an earlier comment where any *str* of zero (eg"0", "000000000000") gives back 0 as the result.  It doesn't follow the other rules on the Lexical Analysis page just as the 'ValueError' for the decintegers isn't on that page.  The lexical analysis page shows that the base 0 is converted to another base according to the first two characters in str, so no other str is evaluated as base 0.  Maybe I should change it to 'a literal 0 evaluates as radix 0'?

I didn't want to use the :attr: on the decinteger.  I just couldn't figure out what it should be in order to link to that definition on the other page.  I had it as a :ref: and that didn't work.  Honestly, I changed it to :attr: when my Travis build failed and I couldn't figure out why.  As you asked, it is supposed to be a reference to the grammar definition.